### DISCUSSION TITLE
Add AST view and Java Element view to SDK?

### DISCUSSION THREAD

**Discussion by iloveeclipse** [eclipse-platform/.github#84] [Ideas]
WDYT about this proposal: https://github.com/eclipse-jdt/eclipse.jdt.ui/issues/387 ?

**Comment by Bananeweizen** [comment]
As software developers we always think that it's easy to find something we created. It isn't (for others than us ourselves). Therefore such debugging views, spies, examples, trace configurations etc. must be promoted much more, and having more of that in the SDK build seems quite good to me.

**Comment by iloveeclipse** [comment]
Done in 4.27
